examples/cassandra/cassandra-controller.yaml

```diff
@@ -1,13 +1,17 @@
 apiVersion: v1
 kind: ReplicationController
 metadata:
-  labels:
-    app: cassandra
   name: cassandra
+  # The labels can be applied automatically
```

Comment:
`can be`  - does this need to be enabled, or is it just what happens?  If this is the default - can be written more precisely - RC can be explicitly defined here, otherwise they are inherited from the pod template.